sdk/appconfiguration/azure-appconfiguration/setup.py

```diff
@@ -60,6 +60,8 @@
         "Development Status :: 5 - Production/Stable",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
```

Comment:
can't comment on the line but I think the python_requires on L73 needs update on the setup.py files